lib/meta-boxes.php

```diff
@@ -47,6 +47,19 @@ function igv_cmb_metaboxes() {
   ) );
 
   $post_meta->add_field( array(
+    'name' => __( 'Autor', 'cmb2' ),
+    'id'   => $prefix . 'post_author',
+    'type' => 'text',
+  ) );
+
+  $post_meta->add_field( array(
```

Comment:
If we are using this date to sort the posts too, why not use published date for all of this?